app/src/main/java/io/apicurio/registry/storage/impl/sql/AbstractSqlRegistryStorage.java

```diff
@@ -2439,7 +2439,7 @@ public void deleteAllUserData() {
         deleteGlobalRules();
 
         handles.withHandleNoException( handle -> {
-            final String tenantId = tenantContext.tenantId();
+//            final String tenantId = tenantContext.tenantId();
```

Comment:
Why comment out & not delete?

Reply:
Whoops.  Didn't mean to leave that comment.  Thanks.